src/daft-logical-plan/src/ops/set_operations.rs

```diff
@@ -4,7 +4,7 @@ use common_error::DaftError;
 use daft_algebra::boolean::combine_conjunction;
 use daft_core::{count_mode::CountMode, join::JoinType, utils::supertype::get_supertype};
 use daft_dsl::{left_col, lit, null_lit, resolved_col, right_col, ExprRef};
-use daft_functions::list::{explode, list_fill};
+use daft_functions_list::{explode, list_fill};
```

Comment:
nit: we could use an import alias in daft_functions to reduce the scope of the pr and files changed, but nit because it's just a refactor technique